openmrs/src/test/java/org/motechproject/openmrs/it/MRSTasksIntegrationBundleIT.java

```diff
@@ -378,32 +412,62 @@ private Task prepareCreatePatientPostActionParameterTask() {
         taskConfig.addAll(taskConfigStepSortedSet);
 
         ArrayList<TaskActionInformation> taskActions = new ArrayList();
-        taskActions.add(prepareCreatePatientActionInformation("Wallstreet 15/2", "John Smith"));
-        taskActions.add(prepareCreatePatientActionInformation("{{pa.0.uuid}}", "Jacob Lee"));
+        taskActions.add(prepareCreatePatientActionInformation("Wallstreet 15/2", "John Smith", false));
+        taskActions.add(prepareCreatePatientActionInformation("{{pa.0.uuid}}", "Jacob Lee", false));
 
         Task task = new Task("OpenMRSPatientPostActionParameterTestTask", triggerInformation, taskActions, taskConfig, true, true);
         getTaskService().save(task);
 
         return task;
     }
 
-    private TaskActionInformation prepareCreatePatientActionInformation(String adress1, String motechId) {
+    private TaskActionInformation prepareCreatePatientActionInformation(String address1, String motechId, Boolean forEncounter) {
         TaskActionInformation actionInformation = new TaskActionInformation("Create Patient [" + DEFAULT_CONFIG_NAME + "]", OPENMRS_CHANNEL_NAME,
                 OPENMRS_CHANNEL_NAME, VERSION, TEST_INTERFACE, "createPatient");
         actionInformation.setSubject(String.format("createPatient.%s", DEFAULT_CONFIG_NAME));
 
         Map<String, String> values = new HashMap<>();
-        values.put(Keys.ADDRESS_1, adress1);
-        values.put(Keys.FAMILY_NAME, "{{ad.openMRS.Patient-" + DEFAULT_CONFIG_NAME + "#0.person.display}}");
-        values.put(Keys.GENDER, "{{ad.openMRS.Patient-" + DEFAULT_CONFIG_NAME + "#0.person.gender}}");
-        values.put(Keys.GIVEN_NAME, "{{ad.openMRS.Patient-" + DEFAULT_CONFIG_NAME + "#0.person.display}}");
+
+        String familyName, gender, givenName;
```

Comment:
I am surprised a bit with this declaration. I think that is not suggested in java, however to me it's ok. 

Can you verify if checkstyle passes when this will be in not test class ?